projects/concentrator/index.js

```diff
@@ -30,8 +31,16 @@ const replacements = [
   "0x73a052500105205d34Daf004eAb301916DA8190f"
 ].map(i => i.toLowerCase())
 
+async function getBalancerLpTvl(balances) {
+  const res = await fetchURL('https://concentrator-api.aladdin.club/data/aladdin/initInfo')
```

Comment:
Please, can u use rpc calls instead of api?

Reply:
Because CTR price not support on coingecko.com

Reply:
U can use defillama pricing api to get price of CTR
https://defillama.com/docs/api